Source/common/SNTFileAccessEvent.m

```diff
@@ -14,6 +14,8 @@
 
 #import "Source/common/SNTFileAccessEvent.h"
 
+#include "Source/common/CertificateHelpers.h"
```

Comment:
```suggestion
#import "Source/common/CertificateHelpers.h"
#import "Source/common/SNTFileAccessEvent.h"
```

Reply:
Updated `CertificateHelpers.h` to be imported, but left the order intact since `SNTFileAccessEvent.h` is the associated header.

https://google.github.io/styleguide/objcguide.html#order-of-includes